osbrain/agent.py

```diff
@@ -714,15 +716,30 @@ def _process_nonsub_event(self, socket_kind, socket, serialized):
         serialized : bytes
             Data received on the socket.
         """
-        message = pickle.loads(serialized)
+        # TODO Instead of identifying the serializer by looking at the message,
+        #      we should have the AgentAddress corresponding to this channel
+        #      as a parameter, in order to get the information from it.
+        #      This will allow us to get rid of the `try-except` clause.
+        # TODO As of now, only `pickle` and `raw` serialization are supported.
```

Comment:
This second TODO can be removed (we only support what is implemented in `AgentAddressSerializer` and, for now, only `raw` and `pickle` are expected).